sdk/identity/azure-identity/tests/test_default.py

```diff
@@ -387,6 +387,21 @@ def validate_tenant_id(credential):
     validate_tenant_id(mock_credential)
 
 
+def test_interactive_browser_client_id():
+    """the credential should allow configuring a client ID for InteractiveBrowserCredential by kwarg"""
+
+    client_id = "client-id"
+
+    def validate_client_id(credential):
+        assert len(credential.call_args_list) == 1, "InteractiveBrowserCredential should be instantiated once"
+        _, kwargs = credential.call_args
+        assert kwargs == {"client_id": client_id, "tenant_id": "72f988bf-86f1-41af-91ab-2d7cd011db47"}
```

Comment:
Suggest validating only that you got the expected client_id, because tenant_id is irrelevant to this feature and varies by environment configuration